src/main/kotlin/no/nav/familie/ef/sak/mapper/SøknadsskjemaMapper.kt

```diff
@@ -92,7 +95,9 @@ object SøknadsskjemaMapper {
                           søktOmSkilsmisseSeparasjon = sivilstandsdetaljer.søktOmSkilsmisseSeparasjon?.verdi,
                           datoSøktSeparasjon = sivilstandsdetaljer.datoSøktSeparasjon?.verdi,
                           separasjonsbekreftelse = tilDomene(sivilstandsdetaljer.separasjonsbekreftelse?.verdi),
-                          årsakEnslig = sivilstandsdetaljer.årsakEnslig?.verdi,
+                          årsakEnslig = sivilstandsdetaljer.årsakEnslig?.let {
+                              EnumTekstverdiMedSvarId(it.verdi, it.svarId ?: MANGLER_SVAR_ID)
+                          },
```

Comment:
Tar veldig gjerne inspill på hvordan vi bør håndtere null-verdier der vi forventer å finne en ```svarId```.

Bør ```svarId``` feltet i ```EnumTekstVerdiMedSvarId´´´ være nullable? og så heller ta null-sjekken der vi faktisk bruker ´´´svarId´´´

Reply:
Kanskje vi skal tillate svarid å være nullable? 
To sider der;

1) kjipt hvis det i 99% av tilfellene ikke er null og vi bruker nullable... 
2) Kjipt hvis vi brekker eksisterende kode med å gjøre den non-nullable og det påvirker oss (vet ikke om det gjør det)